### DISCUSSION TITLE
how to put a transaction ?

### DISCUSSION THREAD

**Discussion by sasdecompte** [nguyenphuminh/JeChain#14] [Q&A] [answered]

**Comment by nguyenphuminh** [comment, marked as answer]
As said in the README, you can send a transaction by using the `sendTransaction` function.

Example:
```js
// Create a new transaction object
const transaction = new Transaction(publicKey, "the address you are sending to", amount, gas);

// sign the transaction
transaction.sign(keyPair);

// broadcast transaction
sendTransaction(transaction);
```
Replace `amount` with the amount of money you want to send, replace `gas` with the amount of gas you want to use, 1 is minimum.

Paste the code in `jenode.js` if you haven't known about this.

**Advanced**

To deploy a contract, replace the recipient's address with `"SC" + code of your contract`.

To add additional arguments when you are calling a contract, add an argument to `Transaction` as an array containing the arguments you want to use.
